### DISCUSSION TITLE
《宝可梦 钻石／珍珠》汉化修正补丁

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/PokemonChineseTranslationRevise#19] [General]
# 《宝可梦 钻石／珍珠》汉化修正补丁

https://blog.www.localhost/PokemonChineseTranslationRevise/DP.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****Yao. 评论于 2021-08-16 10:30:32 (UTC+8)*****

请问一下这个补丁是对着rom直接用还是必须对着YYJoy的汉化补丁用呢？谢谢！

*****Xzonn 回复于 2021-08-19 21:55:57 (UTC+8)*****

对 ROM 使用。

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****B 评论于 2022-07-23 23:49:52 (UTC+8)*****

请问以下，打补丁成功后进游戏后的界面，是不是还叫口袋妖怪？不是精灵宝可梦？

*****这哪我谁 回复于 2022-11-05 13:07:51 (UTC+8)*****

我的也是口袋妖怪……没有预览图好像确实没更图标

**Comment by clavical** [comment]
拿苹果delta模拟器来运行，为啥珍珠钻石的作弊码没用的？
